harvester_e2e_tests/integrations/test_4_vm_backup_restore.py

```diff
@@ -626,6 +626,220 @@ def test_restore_replace_with_vm_shutdown_command(
         )
 
 
+@pytest.mark.p0
+@pytest.mark.backup_target
+@pytest.mark.parametrize(
+    "backup_config", [
+        pytest.param("S3", marks=pytest.mark.S3),
+        pytest.param("NFS", marks=pytest.mark.NFS)
+    ],
+    indirect=True)
+class TestBackupRestoreWithSnapshot:
+
+    @pytest.mark.dependency()
+    def test_connection(self, api_client, backup_config, config_backup_target):
```

Comment:
we already tested the connection in previous class, we don't need duplicate the test case.

Reply:
Thanks for the suggestion. 
The reason to have the test connection running again is to maintain the full function of the backup and restore test cases inside the new test class `TestBackupRestoreWithSnapshot` 

According to the test result in the follow up comment
https://github.com/harvester/tests/pull/1384#issuecomment-2275375643

And consider the trade off of pros and cons, may I keep these tests in the new test class. Many thanks.